app/views/welcome_pages/student.html.erb

```diff
@@ -14,11 +14,19 @@
   <div class="grid md:grid-cols-4 gap-4">
     <div class="col-span-3">
       <p class="body-sm-text mx-2">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut 
-        labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco 
-        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in 
-        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat 
-        non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+        The Department of Psychology provides vehicles for individuals and groups to use for getting to and from their site placements for specific Experiential Learning Courses: 
```

Comment:
Static text needs to be moved to a variable that  an be edited on a per unit basis